### ISSUE TITLE
Unified Webhooks

### ISSUE THREAD

**Issue by jatinsandilya** [revertinc/Revert#133]

**Comment by cmdrGuyson** [comment]
@jatinsandilya Is this issue related to integrating CRM entity subscription webhooks as well?

For example: https://developers.hubspot.com/docs/api/webhooks

**Comment by jatinsandilya** [comment]
@cmdrGuyson Yes. the idea is to have a unified webhook interface for all CRMs (as well as other platforms) 

What made you curious?

**Comment by cmdrGuyson** [comment]
> @cmdrGuyson Yes. the idea is to have a unified webhook interface for all CRMs (as well as other platforms)
> 
> What made you curious?

I was thinking of opening an issue for this specific use case, since we are in need to such webhooks.
